plugins/ruby-puppetdb-foreman/debian/changelog

```diff
@@ -1,3 +1,9 @@
+ruby-puppetdb_foreman (0.1.3-1) stable; urgency=low
```

Comment:
sorry, didn't catch that in the first version... dash instead of underscore is needed here.